### ISSUE TITLE
Unable to load inventory plugin: Ansible 2.7

### ISSUE THREAD

**Issue by yashodhan10** [ansible/ansible#72167]
##### SUMMARY
I'm getting issue while loading the inventory plugin:

```
Populate inventory ------------------------------------------------------ 2.74s
[0;34mUsing ansible.cfg as config file[0m
[1;35m [WARNING]: Failed to load inventory plugin, skipping toml[0m
```

This issue happens intermittently, does not happen when every time I run ansible-playbook. version- 2.7.4

I tried adding this in my config file as well:
```
[inventory]
enable_plugins = host_list, script, auto, yaml, ini
```

Any suggestions? 

##### ISSUE TYPE
- Bug Report

##### COMPONENT NAME

plugin - inventory

##### ANSIBLE VERSION

```paste below
2.7.4
```

##### CONFIGURATION

```paste below

```

##### OS / ENVIRONMENT
RedHat 7.6 (Azure cloud)

##### STEPS TO REPRODUCE

```yaml

```

##### ACTUAL RESULTS

```paste below

```

**Comment by ansibot** [comment]
Files identified in the description:
None

If these files are incorrect, please update the `component name` section of the description or use the `!component` bot command.

[click here for bot help](https://github.com/ansible/ansibullbot/blob/master/ISSUE_HELP.md)

**Comment by ansibot** [comment]
@yashodhan10: Greetings! Thanks for taking the time to open this issue. In order for the community to handle your issue effectively, we need a bit more information.

Here are the items we could not find in your description:
- ansible version
- component name

Please set the description of this issue with an appropriate template from:
https://github.com/ansible/ansible/tree/devel/.github/ISSUE_TEMPLATE

[click here for bot help](https://github.com/ansible/ansibullbot/blob/master/ISSUE_HELP.md)

**Comment by ansibot** [comment]
Files identified in the description:
* [`lib/ansible/plugins/inventory/__init__.py`](https://github.com/ansible/ansible/blob/devel/lib/ansible/plugins/inventory/__init__.py)
* [`lib/ansible/plugins/inventory/advanced_host_list.py`](https://github.com/ansible/ansible/blob/devel/lib/ansible/plugins/inventory/advanced_host_list.py)
* [`lib/ansible/plugins/inventory/auto.py`](https://github.com/ansible/ansible/blob/devel/lib/ansible/plugins/inventory/auto.py)
* [`lib/ansible/plugins/inventory/constructed.py`](https://github.com/ansible/ansible/blob/devel/lib/ansible/plugins/inventory/constructed.py)
* [`lib/ansible/plugins/inventory/generator.py`](https://github.com/ansible/ansible/blob/devel/lib/ansible/plugins/inventory/generator.py)
* [`lib/ansible/plugins/inventory/host_list.py`](https://github.com/ansible/ansible/blob/devel/lib/ansible/plugins/inventory/host_list.py)
* [`lib/ansible/plugins/inventory/ini.py`](https://github.com/ansible/ansible/blob/devel/lib/ansible/plugins/inventory/ini.py)
* [`lib/ansible/plugins/inventory/script.py`](https://github.com/ansible/ansible/blob/devel/lib/ansible/plugins/inventory/script.py)
* [`lib/ansible/plugins/inventory/toml.py`](https://github.com/ansible/ansible/blob/devel/lib/ansible/plugins/inventory/toml.py)
* [`lib/ansible/plugins/inventory/yaml.py`](https://github.com/ansible/ansible/blob/devel/lib/ansible/plugins/inventory/yaml.py)
* [`test/integration/targets/ansible-doc/collections/ansible_collections/testns/testcol/plugins/inventory/statichost.py`](https://github.com/ansible/ansible/blob/devel/test/integration/targets/ansible-doc/collections/ansible_collections/testns/testcol/plugins/inventory/statichost.py)
* [`test/integration/targets/collections/collections/ansible_collections/testns/content_adj/plugins/inventory/statichost.py`](https://github.com/ansible/ansible/blob/devel/test/integration/targets/collections/collections/ansible_collections/testns/content_adj/plugins/inventory/statichost.py)
* [`test/integration/targets/old_style_cache_plugins/plugins/inventory/test.py`](https://github.com/ansible/ansible/blob/devel/test/integration/targets/old_style_cache_plugins/plugins/inventory/test.py)
* [`test/support/integration/plugins/inventory/aws_ec2.py`](https://github.com/ansible/ansible/blob/devel/test/support/integration/plugins/inventory/aws_ec2.py)
* [`test/support/integration/plugins/inventory/docker_swarm.py`](https://github.com/ansible/ansible/blob/devel/test/support/integration/plugins/inventory/docker_swarm.py)
* [`test/support/integration/plugins/inventory/foreman.py`](https://github.com/ansible/ansible/blob/devel/test/support/integration/plugins/inventory/foreman.py)
* [`test/support/integration/plugins/inventory/vmware_vm_inventory.py`](https://github.com/ansible/ansible/blob/devel/test/support/integration/plugins/inventory/vmware_vm_inventory.py)
* [`test/units/cli/test_data/collection_skeleton/plugins/inventory/.git_keep`](https://github.com/ansible/ansible/blob/devel/test/units/cli/test_data/collection_skeleton/plugins/inventory/.git_keep)
* [`test/units/plugins/inventory/__init__.py`](https://github.com/ansible/ansible/blob/devel/test/units/plugins/inventory/__init__.py)
* [`test/units/plugins/inventory/test_constructed.py`](https://github.com/ansible/ansible/blob/devel/test/units/plugins/inventory/test_constructed.py)
* [`test/units/plugins/inventory/test_inventory.py`](https://github.com/ansible/ansible/blob/devel/test/units/plugins/inventory/test_inventory.py)
* [`test/units/plugins/inventory/test_script.py`](https://github.com/ansible/ansible/blob/devel/test/units/plugins/inventory/test_script.py)

If these files are incorrect, please update the `component name` section of the description or use the `!component` bot command.

[click here for bot help](https://github.com/ansible/ansibullbot/blob/master/ISSUE_HELP.md)

**Comment by sivel** [comment]
Ansible 2.7 end of life, and is no longer supported. Please upgrade to a more recent version and validate if this issue is still happening.

**Comment by yashodhan10** [comment]
> Ansible 2.7 end of life, and is no longer supported. Please upgrade to a more recent version and validate if this issue is still happening.

 I did not face this issue again. But when I tried to upgrade to ansible 2.8.16, I got the error on some packages. I use ansible[azure] and after upgrading, it upgraded underlying azure-mgmt packages as well. One package "azure-mgmt-network" was upgraded to version 2.3.0 (which is pretty old) and this version have some issues. There is no "v2018_11_01.models" packge in version 2.3.0. so if someone is using this python package, they'd get the error that module not found. I would strongly recommend to update this baseline dependency of azure-mgmt-network to version 10.0.
